static/flow/views/fullscreen-widget.js

```diff
@@ -0,0 +1,58 @@
+//
+// A widget to make app fullscreen
+//
+var FullScreenWidget = function() {
+
+    var fullscreenContainer = jQuery('#fullscreen-widget');
+    var fullscreenButton  = jQuery('<div>', {id:'fullscreen-button'} );
+    fullscreenButton.appendTo(fullscreenContainer);
+    var fullscreen = false;
+    fullscreenButton.click(function(e) {
+        if (screenfull.enabled) {
+            screenfull.toggle();
+        }
+    });
```

Comment:
I'm not sure I understand the purpose of all of the code below here - shouldn't it be sufficient to have something like 

```
fullscreenButton.click(function(e) {
  if (screenfull.enabled) { 
    screenfull.toggle();
    if (screenfull.isFullscreen) {
      fullscreenButton.addClass("exit");
    } else {
      fullscreenButton.removeClass("exit");
    }
  }
});
```

Reply:
Ah ok I see now that the implementation was mostly taken from SEPA...well, still let me know if you can help me understand what's going on :)

Reply:
I bet there is just something obsolete that I left in in a rush. Let me look.

Reply:
Yup, that was my first implementation before I was redirected to do it the CC way.  Just some obsolete code that was left in.  I removed it.